packages/library-base/src/commonMain/kotlin/io/realm/kotlin/types/RealmSet.kt

```diff
@@ -52,9 +52,16 @@ public interface RealmSet<E> : MutableSet<E>, Deleteable {
      * the elements in a timely manner the coroutine scope will be cancelled with a
      * [CancellationException].
      *
+     * @param keyPaths An optional list of model class properties that defines when a change to
+     * objects inside the set will result in a change being emitted. Nested properties can be
+     * defined using a dotted syntax, e.g. `parent.child.name`. If no keypaths are provided, changes
+     * to all top-level properties and nested properties up to 4 levels down will trigger a change.
+     * Keypaths are only supported for sets of objects.
```

Comment:
why only supported for sets of objects? it's also supported for List, RealmResults etc...